src/routers/posts.ts

```diff
@@ -22,25 +22,1077 @@ import {
 import { postControl, postRangeControl } from '../controllers/postControl';
 const router = express();
 
+/**
+ * @swagger
+ * /posts:
+ *   get:
+ *     tags:
+ *       - Posts operations
+ *     summary: Get all posts
```

Comment:
summary should contains just -
"Must be connected as admin" OR
"Must be connected as user"